git-authors: fix example descriptions (#5987)
tldr-pages/tldr: pages/common/git-authors.md

# git authors

> > Display a list of committers of a Git repository. Everything except `--list` will append the list of commiters to the `AUTHORS` file.

Reviewer: Even tough it does append it, I'm not sure if that's an important detail, since in almost all cases it will create it.

Author: If you have a header in the `AUTHORS` file, I think you don't want to recreate a get a mess with the list.

Reviewer 2: Let's move this note to the `--list` example.
```suggestion
> Generate a list of committers of a Git repository.
```
"Generate" because this isn't actually reading it out of some secret existing file.  It's generating it based on the commits.

Reviewer 3: Generally speaking I think this detail might be best served in the example description. If we move `--list` to be first, then put this in the description of the 2nd, then reword the first to imply it's similar to the 2nd but is slightly different.

> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.

- Append the list of committers to the the `AUTHORS` file and open it in the default editor:

`git authors`

> - Print a full list of committers:

Reviewer 2: ```suggestion
- Print a full list of committers to stdout instead of to the `AUTHORS` file:
```
